addons/binding/org.openhab.binding.harmonyhub/META-INF/MANIFEST.MF

```diff
@@ -23,7 +23,6 @@ Bundle-SymbolicName: org.openhab.binding.harmonyhub;singleton:=true
 Bundle-Vendor: openHAB
 Bundle-Version: 2.4.0.qualifier
 Import-Package: 
- com.google.common.base,
```

Comment:
This binding uses a library (JAR) that depends on this Guava package, see: 

https://github.com/tuck182/harmony-java-client/blob/4ab8d439f5706f4c11ca28f1f14c3f060fda010f/src/main/java/net/whistlingfish/harmony/protocol/OAStanza.java#L13

You might want to check if other bindings also have JARs which use Guave imports.